src/idx.ts

```diff
@@ -9,6 +10,10 @@ declare global {
 
 export function createIDX(ceramic: CeramicApi): IDX {
   const idx = new IDX({ ceramic })
+  window.createTile = createTile
+  window.createDefinition = createDefinition
+  window.exampleDID = 'did:key:z6MkuEd4fm7qNq8hkmWFM1NLVBAXa4t2GcNDdmVzBrRm2DNm'
```

Comment:
Do we need these? If so, seems better to set at the module level than in this function as it's unrelated

Reply:
The window. is just to add it to the dev console for users to query, the functions were needed but are now not being used so they're getting removed. The DID's I would like to keep and this felt like the best spot to expose them for users to query, thinking on it now it makes sense to insert this into our app.ts after authentication. What are your thoughts on this approach?

Reply:
Yes I'm just saying these don't need to be assigned in the `createIDX()` function, they can be set directly in the module as they don't rely on anything from the function call.

Reply:
Sorry I'm not sure my comments are clear, so might be simpler with the code:

```ts
// these don't need to be part of any function call
window.definition = schemas.Definition
window.exampleDID = 'did:key:z6MkuEd4fm7qNq8hkmWFM1NLVBAXa4t2GcNDdmVzBrRm2DNm'
window.exampleDID2 = 'did:3:kjzl6cwe1jw146g16yd2w2dhcirw4zhevar48divve67jx3fq5ztxsx12aimr5y'

export function createIDX(ceramic: CeramicApi): IDX {
  const idx = new IDX({ ceramic })
  // we attach to the Window object here because the IDX instance just got created
  window.idx = idx
  return idx
}
```

Reply:
That is much clearer, thanks!